src/query/vacuum.c

```diff
@@ -2974,10 +2975,19 @@ vacuum_master_task::execute (cubthread::entry &thread_ref)
       return;
     }
 
+  assert (is_active_transaction_server());
```

Comment:
It would be nice to move the first place(2960 line).

Reply:
OK